packages/turf-tin/test.js

```diff
@@ -1,15 +1,24 @@
-var test = require('tape');
-var fs = require('fs');
-var path = require('path');
-var tin = require('./index.js');
+const fs = require('fs');
+const path = require('path');
+const test = require('tape');
+const tin = require('./index.js');
 
-test('tin', function(t){
-  var points = require(path.join(__dirname,  'test', 'Points.json'));
-  var tinned = tin(points, 'elevation');
+const points = require(path.join(__dirname,  'test', 'Points.json'));
 
+test('tin - z property', t => {
+  const expected = require(path.join(__dirname,  'test', 'Tin.json'));
+  const tinned = tin(points, 'elevation');
   t.equal(tinned.features[0].geometry.type, 'Polygon');
   t.equal(tinned.features.length, 24);
+  t.deepEqual(tinned, expected, 'tinned polygons match');
+  t.end();
+});
 
-  fs.writeFileSync(path.join(__dirname, 'test', 'Tin.json'), JSON.stringify(tinned));
```

Comment:
Saving the outputs is very useful to see the final results, also saving the fixtures using `.geojson` makes it easier to preview the GeoJSON directly in GitHub.